sublime-package.json

```diff
@@ -10,6 +10,14 @@
 					"definitions": {
 						"PluginConfig": {
 							"properties": {
+								"initializationOptions": {
+									"type": "object",
+									"properties": {
+										"settings": {
+											"$ref": "sublime://settings/LSP-lemminx#/definitions/PluginConfig/properties/settings"
```

Comment:
I thought you can only reference keys defined directly under `definitions` but it seems to work this way.

I would probably still make `settings` into each own definition but I won't insist.

Reply:
Had this in mind, too, was not sure about the naming though. Maybe `PluginSettings` or `ServerSettings`?

Reply:
The root object is called PluginConfig so something in that vein, for consistency.

Reply:
ServerSettings would work for me too